### ISSUE TITLE
Fails when tsconfig is not set

### ISSUE THREAD

**Issue by skrat** [fuse-box/fuse-box-typechecker#56]
https://github.com/fuse-box/fuse-box-typechecker/blob/36263dd6bb05f42073e94b187bb43915c2948ab9/src/index.ts#L51

**Comment by vegarringdal** [comment]
@skrat 
You feel tsConfig should be optional too?

```ts
// get typechecker
const { TypeChecker } = require('fuse-box-typechecker');

const testSync = TypeChecker({
    tsConfig: './tsconfig.json',
    basePath:'./',
    tsLint:'./tslint.json', //optional
    name: 'Test Sync'
    // for more option, see ITypeCheckerOptionsInterface in bottom on readme
})

// call function
testSync.XXXXXXXX() //see public functions
```

**Comment by skrat** [comment]
Oh sorry, I thought it is but it's just the tslint. But yeah, a default would be nice.

**Comment by vegarringdal** [comment]
Might be a nice feature to have it as a optional/default settings.
If you have any suggestions then tell me. Ill try and have a look at it during the weekend :-)

**Comment by vegarringdal** [comment]
@skrat 

Added this as optional in version
`2.12.0-next.1`

You you give it a test to see it works in your end?

**Comment by vegarringdal** [comment]
fix in 2.12.0 😄
